configure.ac

```diff
@@ -14,8 +14,6 @@ AM_CONFIG_HEADER(config.h)
 AC_CONFIG_MACRO_DIR([m4])
 AM_MAINTAINER_MODE
 AC_SUBST(SHARED_VERSION_INFO)
-# need shared libraries for python interfaces
-AM_ENABLE_SHARED
```

Comment:
You can change this to `AC_ENABLE_SHARED(no)` to make the default static-only (you can still do `--enable-shared`), and you'll need to change Travis to use that flag.   But I think we should leave it as `AC_ENABLE_SHARED` (building shared libraries) and just disable Python...